src/go/rdctl/pkg/config/config.go

```diff
@@ -51,17 +54,24 @@ var (
 	configDir         string
 	configPath        string
 	defaultConfigPath string
+	configFile        = "rd-engine.json"
+	rdSubDir          = "rancher-desktop"
 )
 
 // DefineGlobalFlags sets up the global flags, available for all sub-commands
 func DefineGlobalFlags(rootCmd *cobra.Command) {
 	var err error
-
-	configDir, err = os.UserConfigDir()
-	if err != nil {
-		log.Fatal("Can't get config-dir: ", err)
+	if runtime.GOOS == "linux" && isWSLDistro() {
+		if defaultConfigPath, err = wslifyConfigPath(); err != nil {
+			log.Fatalf("Can't get config-dir: %v", err)
+		}
+	} else {
+		configDir, err = os.UserConfigDir()
+		if err != nil {
+			log.Fatalf("Can't get config-dir: %v", err)
+		}
+		defaultConfigPath = filepath.Join(configDir, rdSubDir, configFile)
```

Comment:
This line can stay outside the conditional, as discussed above. You also don't need symbolic names for `rdSubDir` and `configFile` anymore, since the constants are only used once.